test/conformance/api/v1/service_test.go

```diff
@@ -237,6 +241,42 @@ func TestServiceBYOName(t *testing.T) {
 	}
 }
 
+// TestRunLatestServiceForPodAutoscalerAnnotation creates a Service with invalid class annotation for autoscaler.
+func TestRunLatestServiceForPodAutoscalerAnnotation(t *testing.T) {
+	t.Parallel()
+	clients := test.Setup(t)
+	names := test.ResourceNames{
+		Service: test.ObjectNameForTest(t),
+		Image:   test.PizzaPlanet1,
+	}
+	// Clean up on test failure or interrupt
+	defer test.TearDown(clients, names)
+	test.CleanupOnInterrupt(func() { test.TearDown(clients, names) })
+	// Setup initial Service
+	_, err := v1test.CreateService(t, clients, names, rtesting.WithConfigAnnotations(map[string]string{
+		autoscaling.ClassAnnotationKey: "hpa1", //Invalid class annotation other than KPA and HPA
```

Comment:
Things involving annotations don't belong in conformance.

Reply:
Thank you. 
Removed :+1: